include/hpp/constraints/tools.hh

```diff
@@ -192,6 +192,7 @@ void JlogSE3(const Transform3s& M, Eigen::MatrixBase<Derived> const& Jlog) {
   const vector3_t& p = M.translation();
   value_type theta;
   vector3_t r;
+  r.setZero();
```

Comment:
Is this for anything else than a compiler warning ?

Reply:
Indeed, it only fixes a compilation warning.